src/lib/layout/hexagons/TeamMemberHexagon.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+  import Hexagon from './Hexagon.svelte';
+  import Icon from '$lib/component/Icon.svelte';
+  import Icons from '$lib/component/Icons';
+  import type { TeamMember } from '@/model/TeamMember';
+
+  export let teamMember: TeamMember;
+</script>
+
+<Hexagon orientation="horizontal">
+  <div class="group relative" data-testid="hexagon">
+    <div
+      class="
+       absolute top-full z-20 w-full -translate-y-16 px-4 pb-4 duration-500 group-hover:top-1/2 group-hover:-translate-y-1/2"
+    >
+      <p
+        class="mx-auto text-center text-sm font-bold leading-tight text-gray-100 transition-all sm:text-sm md:text-sm lg:text-base xl:text-xl"
+      >
+        {teamMember.name.split(' ')[0]}
+        <br />
+        {teamMember.name.split(' ')[1]}
+      </p>
+      <p
+        class="mx-auto
+        text-center text-xs leading-tight text-gray-100 opacity-0 transition-all duration-500 ease-out group-hover:opacity-100 sm:text-xs md:text-sm lg:text-base xl:text-lg"
+      >
+        {teamMember.role}
+      </p>
+      <div class="mt-1 flex justify-center space-x-1">
+        {#if teamMember.linkedin}
+          <a
+            href={teamMember.linkedin}
+            class="opacity-0 transition-all duration-500 ease-out group-hover:opacity-100"
+            aria-label="{teamMember.name}'s LinkdIn"
+          >
+            <Icon src={Icons.Linkedin} color="white" size="2.5vw" /></a
+          >
+        {/if}
+        {#if teamMember.gitHub}
+          <a
+            href={teamMember.gitHub}
+            class="opacity-0 transition-all duration-500 ease-out group-hover:opacity-100"
+            aria-label="{teamMember.name}'s GitHub"
+            ><Icon src={Icons.Github} color="white" size="2.5vw" /></a
+          >
```

Comment:
Using viewport width here does make the icons _really_ small on mobile. We could try to use percentage-based widths, but since the icon's parent is the anchor tag, it's not going to make a difference. 
However, what if you don't try to modify the size of the icon, but instead change its parent's size and pass "100%" to the icon's width? Like so:
```suggestion
        {#if teamMember.gitHub}
          <a
            href={teamMember.gitHub}
            class="w-[20%] opacity-0 transition-all duration-500 ease-out group-hover:opacity-100"
            aria-label="{teamMember.name}'s GitHub"
            ><Icon src={Icons.Github} color="white" size="100%" /></a
          >
```

What this does is adapt the width of the anchor tag to 20% of the width of the parent div around it (the horizontal container holding all of the icons) and increase the width of the icon to occupy the whole anchor tag. This will preserve the ratio between container/icon sizes and make the icons a lot bigger at smaller viewports. Here's how it looks when applied to all icons:

Desktop
![image](https://github.com/NIAEFEUP/website-niaefeup-frontend/assets/75742355/4305c030-8795-4299-905c-52f64c3cbe63)

Small Mobile
![image](https://github.com/NIAEFEUP/website-niaefeup-frontend/assets/75742355/d2cef6f6-9c79-482a-a0bb-e15e5ef8e28d)

Very Small Viewport
![image](https://github.com/NIAEFEUP/website-niaefeup-frontend/assets/75742355/85abd9cc-4b1c-4da7-b8e4-64dd71d17f3c)

It seems to work fine with the specified percentages, but feel free to experiment a bit more.

Reply:
This worked really well, thank you for the suggestion!